app/src/scan.py

```diff
@@ -55,10 +57,35 @@ class DMARCScanResult:
     tags: Dict[str, Any] = field(default_factory=dict)
 
 
+@dataclass
+class TLSServerScanResult:
+    host: str
+    port: int
+    connected: bool
+    is_certificate_valid: Optional[bool]
+    supported_protocols: List[str]
+
+
+@dataclass
+class TLSMailTransportScanResult:
+    # mta_sts = None
+    mail_exchanges: List[TLSServerScanResult]
+    valid = True
+    errors: List[str]
+    warnings: List[str]
+
+
+@dataclass
+class TLSMailScanResult:
+    mail_transfer: Optional[TLSMailTransportScanResult]
+    # mail_provider = None
```

Comment:
don't leave commeted code

Reply:
I wanted to explain why there is a `TLSMailScanResult` class with only one element. It is because there will be other things there in the future!